language/move-prover/tests/sources/functional/address_serialization_constant_size.exp

```diff
@@ -0,0 +1,15 @@
+Move prover returns: exiting with boogie verification errors
+error:  A postcondition might not hold on this return path.
+
+    ┌── tests/sources/functional/address_serialization_constant_size.move:17:9 ───
+    │
+ 17 │         ensures len(LCS::serialize(u1)) == len(LCS::serialize(u2));
```

Comment:
If a function is SUPPOSED to get an error, suffix the name with _incorrect so people don't get worried when they see it.

The u64 example might be confusing, because (I think) u64's actually SHOULD always serialize to the same length. You just didn't include that fact in the model, so they fail here as expected.  Perhaps you should use a different type (e.g., a vector<u8>?) which really could serialize to different lengths.

Reply:
Yes, good points.